src/model/Limits.java

```diff
@@ -0,0 +1,37 @@
+package model;
+
+public class Limits {
+  private double upper;
+  private double lower;
+
+  public Limits(double upper, double lower){
+    this.upper = upper;
+    this.lower = lower;
+  }
+
+  public void setLower(double lower) {
```

Comment:
make it throw maybe exception so that we can put that exception into an errorlabel

Reply:
exeptions are added